docs/data_management/research_data_responsibilities.md

```diff
@@ -26,7 +26,11 @@ Researchers and users of Cheaha are responsible to organize data, archive inacti
 
 UAB IT has a process for granting access to data of former researchers or collaborators who are no longer with the institution. This process ensures compliance with regulatory protocols.
 
-To request access to data of former UAB user, the first step is to fill out the [Third-Party Data Access form](https://uabprod.service-now.com/service_portal?id=sc_cat_item&sys_id=bd3721e2374c27c0daa253b543990e5d). In the “justification/description” field specify that you are requesting access to data for `<BlazerId>` on GPFS at the Research Computing System. Once submitted this form, a ticket is created and routed to the appropriate reviewers for authorization.
+To request access to data of former UAB user, the first step is to fill out the [Third-Party Data Access form](https://uabprod.service-now.com/service_portal?id=sc_cat_item&sys_id=bd3721e2374c27c0daa253b543990e5d). We highly recommend requesting access to all three private storage areas. Consider adding the following to the "Justification/Description" box in the security exception form as you fill it out. Replace `<BlazerID>` with the BlazerID of the user you are requesting access for.
+
+`Please grant access to "/data/user/", "/home/", "/scratch/" directories of <BlazerID>.`
```

Comment:
Instead of verbatim backticks, let's use blockquote

```
This is normal text.

> This is a block quote.
```

This is normal text.

> This is a block quote.

Reply:
Or we can use a fenced code environment. This will give them a copy button.

```
Please grant access to "/data/user/", "/home/", "/scratch/" directories of <BlazerID>.
```